htdocs/core/tpl/extrafields_list_search_sql.tpl.php

```diff
@@ -41,7 +41,7 @@
 			if (in_array($typ, array('chkbxlst', 'checkbox'))) $mode_search=4;	                            // Search on a multiselect field with sql type = text
 			if (is_array($crit)) $crit = implode(' ', $crit); // natural_search() expects a string
 			elseif ($typ === 'select' and is_string($crit) and strpos($crit, ' ') === false) {
-				$sql .= ' AND (' . $extrafieldsobjectprefix.$tmpkey . ' = \'' . $db->escape($crit) . '\')';
+				$sql .= " AND (" . $extrafieldsobjectprefix.$tmpkey . " = '" . $db->escape($crit) . "')";
```

Comment:
Hello,

This modification breaks compat with PostgreSQL. We can't use double quote to compare strings, for more information : https://www.postgresql.org/docs/12/sql-syntax-lexical.html

Best regards,
L.

Reply:
This fix exactly what you say. Before it was using double quotes, after it uses simple quotes.

Reply:
sorry for the mistake... I rode the commit log too fast.